src/common/json.cpp

```diff
@@ -25,6 +26,7 @@ namespace json
 
     void to_file(std::wstring_view file_name, const JsonObject& obj)
     {
-        std::wofstream{ file_name.data(), std::ios::binary } << obj.Stringify().c_str();
+        std::wstring obj_str{ obj.Stringify().c_str() };
+        std::ofstream{ file_name.data(), std::ios::binary } << winrt::to_string(obj_str);
```

Comment:
Should we check if file is open here as well ?

Reply:
It doesn't throw if a file isn't open, but we could check and return `false` in that case.